src/main/java/org/traccar/database/IdentityManager.java

```diff
@@ -26,6 +26,10 @@
 
     Device getByUniqueId(String uniqueId) throws Exception;
 
+    String getDevicePassword(long id);
+
+    String getDevicePassword(long id, String defaultPassword);
```

Comment:
What's the reason for having two versions of `getDevicePassword`?

Reply:
Nothing special. Keeping just one. Thanks.